### ISSUE TITLE
"Login attempt failed"

### ISSUE THREAD

**Issue by bg1000** [maykar/lovelace-swipe-navigation#16]
Installed with HACS under HA 0.104.3 to use with the official android app.  Configuration in the raw config editor is as follows:

```
resources:
  - type: js
    url: /community_plugin/lovelace-swipe-navigation/swipe-navigation.js
swipe_nav:
  wrap: true
  animate: swipe
  skip_tabs: '1,2,3,4,5'
  prevent_default: true
  swipe_amount: 20
```

Swiping works but causes a persistent notification stating "Login attempt failed" - IP indicates the attempt is coming from the phone.

**Comment by maykar** [comment]
Using the Official Android app and your config and I can't replicate this issue.

Can you share your whole resources section (if you have more to it than what you've shared), so that I can test with some of the other addons you may have?

Can you confirm that you don't receive these errors if you temporarily remove swipe-navigation from your resources?

**Comment by bg1000** [comment]
Sure. 

```
resources:
  - type: module
    url: /community_plugin/simple-thermostat/simple-thermostat.js
  - type: js
    url: /community_plugin/lovelace-swipe-navigation/swipe-navigation.js
swipe_nav:
  wrap: false
  animate: swipe
  skip_tabs: '1,2,3,4,5'
  prevent_default: true
  swipe_amount: 20
```

So the only custom components I have installed are HACS itself and the two listed above.  I've been running the thermostat for about a week.  The login thing seemed, to start when I installed swipe-navigation last night. The notification usually appears right after I swipe. I'm running HASS in docker v 0.104.3.  I'm happy to provide more information or try some things on my installation if that would help.

**Comment by maykar** [comment]
Does this only happen when using the swipe animation? `animate: swipe`

**Comment by bg1000** [comment]
I spent some more time trying different things and realized this isn't being caused by swipe-navigation.  I have a picture entity card streaming a camera that is currently disconnected.  It shows the error message only when that tab is selected.  It wasn't happening before I installed swipe-navigation because I wasn't clicking on it.  Once I installed swipe-navigation and started swiping I would bring up this tab which would cause the error.  I deleted the picture entity card the error went away.  I'm sorry for raising this as an issue when it wasn't your issue at all.  Closing.
